fe/fe-core/src/main/java/org/apache/doris/nereids/parser/ParseDialect.java

```diff
@@ -29,7 +29,11 @@ public enum ParseDialect {
     /**
      * Doris parser and it's version is 2.0.0.
      */
-    DORIS_2_ALL(Dialect.DORIS, Version.DORIS_2_ALL);
+    DORIS_2_ALL(Dialect.DORIS, Version.DORIS_2_ALL),
+    /**
+     * Hive parser.
+     */
+    HIVE_ALL(Dialect.HIVE, Version.HIVE_ALL);
```

Comment:
Use HIVE_ALL, the high and low version hive dialect is compatible? should split them to hive2 , hive3?